manifests/lib/python.pp

```diff
@@ -1,5 +1,10 @@
-# This class installs the python libs for postgresql. See README.md for more
-# details.
+# @summary This class installs the python libs for postgresql.
+#
+# @param package_name
+#  String. The name of the PostgreSQL Python package.
+# @param package_ensure
+#  Enum['present','absent']
```

Comment:
Same as before. The data type itself needs changed to reflect this. Also needs description.